src/components/CodeOverview/index.tsx

```diff
@@ -48,7 +48,7 @@ export class CodeOverviewBase extends React.Component<Props, State> {
   static defaultProps = {
     _debounce: debounce,
     _window: window,
-    initialOverviewRef: React.createRef<HTMLDivElement>(),
```

Comment:
Ah, so creating a ref here means that every instance of this component will share a ref? I would have thought that each one would get its own ref, because instances of components don't share props, do they? Is it something about the nature of a ref, or `React.createRef`?

Reply:
> Ah, so creating a ref here means that every instance of this component will share a ref? 

Yes

> I would have thought that each one would get its own ref, because instances of components don't share props, do they? Is it something about the nature of a ref, or `React.createRef`?

It's because the `static` keyword of `static defaultProps` creates a class property, not an instance property. 

I was experimenting with auto-scrolling content in `AccordionItem` using the same ref setup. With this bug, the last item to be rendered was the only one that got scrolled. This makes sense because they were all pointing to the same ref variable; the last one overwrote the others.